src/Directory.build.props

```diff
@@ -30,7 +30,11 @@
     </When>
   </Choose>
 
-  <ItemGroup>
+  <ItemGroup Condition="$(IsTestProject)">
+    <PackageReference Include="xunit" Version="2.3.1" />
+    <PackageReference Include="xunit.runner.console" Version="2.3.1" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.3.1" />
+    <PackageReference Include="Xunit.StaFact" Version="0.2.9" />
     <PackageReference Include="Nerdbank.GitVersioning" Version="2.1.23" PrivateAssets="all" />
```

Comment:
I don't think this should be in the `IsTestProject` item group. Otherwise, versioning doesn't seem work in the branch.